### PR TITLE
update readme in packages

### LINKED ISSUE #2487: Update readmes of deprecated packages
A task from https://github.com/tinacms/tinacms/issues/2466

### PR DESCRIPTION
**Pull request by logan-anderson**
Fixes #2487

Would love it if someone could check this over for spelling / grammar / tone issues. 

Thank you 😄

### CODE REVIEW

**Review by perkinsjr — DISMISSED** [review]
We should offer a solution:

Warning: This package is deprecated and no longer supported or receiving updates. Looking to get started with Tina?  Check out our getting started guide LINK TO getting started

File: packages/next-tinacms-github/README.md [hunk 1]
```diff
@@ -1,230 +1,7 @@
 # _next-tinacms-github_
 
-This package provides helpers for managing the **GitHub auth token** for requests, as well as
-providing helpers for **loading content from the Github API**.
+# DEPRECATION WARNING
 
-## Installation
+> Warning: We are no longer supporting this package, it is deprecated and will not receive further updates
```
[the remark this answers is not in the document]
Comment (logan-anderson) [reply packages/next-tinacms-github/README.md:5]: Yeah for sure! I currently have a link to the docs. Would this be better linked to a get started guide? maybe https://tina.io/docs/setup-overview/

[the remark this answers is not in the document]
Comment (perkinsjr) [reply packages/next-tinacms-github/README.md:5]: Yup that would be a good place. sorry I moved this to an actual change after the fact 👍

**Comment by logan-anderson** [comment]
> We should offer a solution:
> 
> Warning: This package is deprecated and no longer supported or receiving updates. Looking to get started with Tina? Check out our getting started guide LINK TO getting started

@perkinsjr updated to add a link to the get started guide.

**Review by jamespohalloran — APPROVED** [review]
This looks good to me!

**Pull request merged**